modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/ShareFragment.kt

```diff
@@ -4,124 +4,264 @@ import android.os.Bundle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.DpSize
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import androidx.fragment.app.activityViewModels
+import androidx.lifecycle.asFlow
 import au.com.shiftyjelly.pocketcasts.analytics.AnalyticsTrackerWrapper
 import au.com.shiftyjelly.pocketcasts.analytics.SourceView
+import au.com.shiftyjelly.pocketcasts.compose.AppTheme
+import au.com.shiftyjelly.pocketcasts.compose.theme
+import au.com.shiftyjelly.pocketcasts.models.entity.BaseEpisode
+import au.com.shiftyjelly.pocketcasts.models.entity.Podcast
 import au.com.shiftyjelly.pocketcasts.models.entity.PodcastEpisode
-import au.com.shiftyjelly.pocketcasts.player.databinding.FragmentShareBinding
 import au.com.shiftyjelly.pocketcasts.player.viewmodel.PlayerViewModel
 import au.com.shiftyjelly.pocketcasts.repositories.podcast.SharePodcastHelper
-import au.com.shiftyjelly.pocketcasts.repositories.podcast.SharePodcastHelper.ShareType
 import au.com.shiftyjelly.pocketcasts.ui.helper.StatusBarColor
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
 import au.com.shiftyjelly.pocketcasts.views.extensions.applyColor
 import au.com.shiftyjelly.pocketcasts.views.fragments.BaseDialogFragment
 import com.google.android.material.bottomsheet.BottomSheetBehavior
 import com.google.android.material.bottomsheet.BottomSheetDialog
 import dagger.hilt.android.AndroidEntryPoint
 import io.reactivex.disposables.Disposable
 import javax.inject.Inject
+import au.com.shiftyjelly.pocketcasts.localization.R as IR
 
 @AndroidEntryPoint
 class ShareFragment : BaseDialogFragment() {
 
     @Inject lateinit var analyticsTracker: AnalyticsTrackerWrapper
     override val statusBarColor: StatusBarColor? = null
-
+    private lateinit var composeView: ComposeView
     private val viewModel: PlayerViewModel by activityViewModels()
-    private var binding: FragmentShareBinding? = null
     private var disposable: Disposable? = null
-
     override fun onPause() {
         super.onPause()
 
         disposable?.dispose()
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        val binding = FragmentShareBinding.inflate(inflater, container, false)
-        this.binding = binding
-
-        val podcast = viewModel.podcast
-        val episode = viewModel.episode
+        return ComposeView(requireContext()).also {
+            composeView = it
+        }
+    }
 
-        binding.buttonSharePodcast.setOnClickListener {
-            if (podcast != null) {
-                SharePodcastHelper(
-                    podcast,
-                    null,
-                    null,
-                    requireContext(),
-                    ShareType.PODCAST,
-                    SourceView.PLAYER,
-                    analyticsTracker
-                ).showShareDialogDirect()
+    @Suppress("DEPRECATION")
```

Comment:
`onActivityCreated` has been deprecated. Should we consider to use `OnViewCreated` function? wdyt?